### PR TITLE
Added IPSO objects

### PR DESCRIPTION
**Pull request by akeranen**

### CODE REVIEW

File: odmObject/odmobject-accelerometer.sdf.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
+  "info": {
+    "title": "OMA LwM2M Accelerometer (Object ID 3313)",
+    "version": "20191002",
+    "copyright": "Copyright (c) 2018 IPSO.",
```
Comment (WAvdBeek) [inline odmObject/odmobject-accelerometer.sdf.json:5]: also 2019?
(for all objects..)

File: odmObject/odmobject-accelerometer.sdf.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    "license": "https://github.com/one-data-model/oneDM/blob/master/LICENSE"
+  },
+  "namespace": {
```
Comment (WAvdBeek) [inline odmObject/odmobject-accelerometer.sdf.json:8]: i thought we decided that odm is the default name space..
e.g. this namespace declaration should not be there

File: odmObject/odmobject-ipso-temperature.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
+{
```
Comment (WAvdBeek) [inline odmObject/odmobject-ipso-temperature.json:1]: can we remove this one from this pull request, and add changes to the existing OCF temperature?

File: odmObject/odmobject-ipso-humidity.json [hunk 4]
```diff
@@ -0,0 +1,65 @@
+{
```
Comment (WAvdBeek) [inline odmObject/odmobject-ipso-humidity.json:1]: can we remove this file from this pull request and add the changes to the OCF humidity?

File: odmObject/odmobject-ipso-temperature.json (hunk 3 above)
Comment (nwidell) [reply odmObject/odmobject-ipso-temperature.json:1]: I propose to keep both for now (it is the playground after all), in order to look at how to do both odm adoption process and how the handle mappings file. E.g. for temperature the differences how range is indicated.

File: odmObject/odmobject-accelerometer.sdf.json (hunk 1 above)
Comment (akeranen) [reply odmObject/odmobject-accelerometer.sdf.json:5]: Good point. I simply copied this from the source file but indeed now it should contain also 2019.

File: odmObject/odmobject-accelerometer.sdf.json (hunk 2 above)
Comment (akeranen) [reply odmObject/odmobject-accelerometer.sdf.json:8]: True, for ODM Objects it should not be there. For native IPSO objects it should be. Right now the files are kind of both. But I'll re-generate versions without.

**Pull request merged**